.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  extends: 'nava',
+  env: {
+    mocha: true
+  }
+}
```

Comment:
Just for my own learning -- does the linter automatically get run when you do `npm test` if you include this file? Or how does that auto-running of the linter work?